### ISSUE TITLE
Question and Request

### ISSUE THREAD

**Issue by Ravvit528** [GoldieLeGenie/Dragon-quest-3HD2D-Randomizer#4]
Hi, how are you
I have one question, but I don't know where to ask, so I'll write it down here.

Could you please make a MOD "Status does not increase when level up"?

I would like to use your wonderful random MOD to try the game, "There is no status increase at the level, but instead, use random skills, seeds, and other items to capture the game."

I am sorry that I have sent you so many requests and messages so far,
I would be very happy if you could take time to make this MOD

**Comment by GoldieLeGenie** [comment]
Hello so , you want when you level up you keep the same stats ? Exemple at lvl 30 you have the same stats at lvl 1 ?

**Comment by Ravvit528** [comment]
yes, I want to use your Randomize Leaning Skills to learn only skills even if the level is 40 but the status is the same as level 1

**Comment by GoldieLeGenie** [comment]
Alright should be easy to do 👍

**Comment by Ravvit528** [comment]
Thank you from the bottom of my heart☺️

**Comment by Ravvit528** [comment]
Hello,
You have released the latest version of MOD many times and it is very wonderful.
I want to play Ver.0.5, but I am looking forward to playing with the MOD I requested the other day.
Do you think it will take a little longer to complete the MOD?
I don't mean to rush you, but I'm really looking forward to it, so I asked a question :)

**Comment by GoldieLeGenie** [comment]
Hey i was working on the randomized spells of the monsters, which is why I haven't released the mod you wanted yet. As soon as I finish, I'll add it 👍

**Comment by Ravvit528** [comment]
hello!
Can you teach me How to Unpak game files?
I was try find how to unpak, but I didn't it...
I need All Japanese text files:)
